yt_dlp/extractor/orf.py

```diff
@@ -565,3 +567,71 @@ def _real_extract(self, url):
             })
 
         return self.playlist_result(entries)
+
+
+class ORFONIE(InfoExtractor):
+    _VALID_URL = r'https?://on\.orf\.at/video/(?P<id>\d{8})/(?P<slug>[\w-]+)'
+    _TESTS = [{
+        'url': 'https://on.orf.at/video/14210000/school-of-champions-48',
+        'info_dict': {
+            'id': '14210000',
+            'ext': 'mp4',
+            'duration': 2651.08,
+            'thumbnail': 'https://api-tvthek.orf.at/assets/segments/0167/98/thumb_16697671_segments_highlight_teaser.jpeg',
+            'title': 'School of Champions (4/8)',
+            'description': 'md5:d09ad279fc2e8502611e7648484b6afd',
+            'media_type': 'episode',
+        }
+    }]
+
+    def _call_api(self, video_id, display_id):
+        # NOTE: the prefix `3dSlfek03nsLKdj4Jsd` is only based on my observation on several
+        # api call. This string may change in future
```

Comment:
The comment is not necessary imo.
Also, this method does quite a bit more than just call the api, it could be renamed, e.g. `_extract_video`

Reply:
Resolved in https://github.com/yt-dlp/yt-dlp/pull/9113/commits/a857660b61fae942a1b05dfd29326e33d203f0f3 and https://github.com/yt-dlp/yt-dlp/pull/9113/commits/3944e4a692a361c6f162eee5b67c7f2766b85380